### ISSUE TITLE
Relative default image path with filename not working

### ISSUE THREAD

**Issue by sam9032** [Zettlr/Zettlr#3199]
Hey there!
First things first thank you so much for this piece of software! This is a gorgous example how good a open source program can be. Usually electron apps just doesn't work well in a lot of details on linux systems but your app is just way better than the usual. Just one example is the dark/light theme implementation, which switches as soon as the OS settings change. Right now I didn't find another electron app who accomplishes that on linux. Thanks! :100: 

Now back to my issue.. If there is already a similar issue, please excuse my duplicate, but I didn't find one.

In the preferences setting there is under "editor" the entry for the "Default image path":

![image](https://user-images.githubusercontent.com/12381166/153892349-ee6434aa-2c7a-4a43-95d1-26463d81bc50.png)
 
If I set it to a relative folder `./${filename}.assets`, then I would expect Zettlr to create the folder with the real filename, like `realfilename.assets`. But instead it creates the folder with the name `${filename}.assets`, without respecting the variable..

![image](https://user-images.githubusercontent.com/12381166/153893697-72d16f6b-dc5e-4ebd-9197-31b1331ed085.png)

Is there a way, to get the real filename on that folder without manually creating it?

Best regards
Sam

**Comment by nathanlesage** [comment]
Not yet! But it could be possible!

**Comment by sam9032** [comment]
Oh interesting, I thought the entry came as the default at some point. Or is it the default on another OS?

**Comment by nathanlesage** [comment]
No, unfortunately we have to implement any magic by ourselves :(

**Comment by sam9032** [comment]
I changed the title, since the feature is not implemented at all. So just a unimportant feature request. Thank you!

**Comment by adamprada** [comment]
I would like to also request this. Having a specific folder for each markdown file makes it very easy to send anyone a self-contained document (e.g. by zipping the .md together with its directory). It extermely adds to the utility of the editor.

**Comment by gynamics** [comment]
emm... #485 metioned this feature, it seems that there was a simple implement but dropped in following updates. However, that work is still a easy piece. Why not just force to match string "${filename}" as a special name? We actually do not need more true variables here. And this feature is absolutely not "something unimportant", I would rather write raw markdown without this.
